Source/WebKit/WebProcess/Plugins/PDF/PDFPluginBase.mm

```diff
@@ -148,6 +148,11 @@
     m_view->frame()->document()->loader()->addArchiveResource(resource.releaseNonNull());
 }
 
+void PDFPluginBase::geometryDidChange(const IntSize& pluginSize, const AffineTransform& pluginToRootViewTransform)
```

Comment:
is the compiler going to complain here about unused parameter?

Reply:
Thankfully WebKit doesn't enable that silly warning